### DISCUSSION TITLE
Yarn Berry PnP + Quasar Error

### DISCUSSION THREAD

**Discussion by zN3utr4l** [coreybutler/nvm-windows#1115] [Support] [answered]
I opened an issue on the quasar repository, but I don't know if the error is due to quasar.

Could you take a look?

https://github.com/quasarframework/quasar/discussions/17012

**Comment by zN3utr4l** [comment]
> I use **nvm windows** [https://github.com/coreybutler/nvm-windows].
> 
> _**I don't understand why it tells me that I don't have corepack enable, even though I activated it from powershell with admin privileges.**_
> 
> ```
> PS G:\My\My.Hmi\My.Hmi> corepack --version
> 0.23.0
> PS G:\My\My.Hmi\My.Hmi> yarn set version berry
> ➤ YN0000: You don't seem to have Corepack enabled; we'll have to rely on yarnPath instead
> ➤ YN0000: Downloading https://repo.yarnpkg.com/4.1.1/packages/yarnpkg-cli/bin/yarn.js
> ➤ YN0000: Saving the new release in .yarn/releases/yarn-4.1.1.cjs
> ➤ YN0000: Done with warnings in 0s 245ms
> ```
> 


I think this part of the issue is nvm's responsibility or not?

**Reply by coreybutler** [reply, marked as answer]
corepack may be enabled in your node version, but is it configured for the project in your package.json?

https://nodejs.org/api/corepack.html#configuring-a-package

NVM4W merely installs Node and its features... then it backs off and leaves your environment alone. In other words, _using_ corepack doesn't interact with NVM4W at all. If you've verified the package.json, I'd recommend opening an issue in the corepack repo for more specific support.
